includes/Hasher/DbRevisionVerificationRepo.php

```diff
@@ -7,14 +7,24 @@
 use Wikimedia\Rdbms\ILoadBalancer;
 
 class DbRevisionVerificationRepo implements RevisionVerificationRepo {
+	/**
+	 * @var ILoadBalancer 
+	 */
```

Comment:
This is negative value. The info is already there. So the comment is noise at best, and potentially something that got out of date and misleads.

Reply:
wont the codesniffer fail on missing docs for variables docs once the restrictions are removed in .phpcs.xml?

Reply:
Then don't remove the restrictions. Why follow rules that force you to do counterproductive things?

Reply:
I take it that this has nothing to do with PHP 7.4 compatibility, but simply a config in .phpcs.xml? What is the name of the restriction?

Reply:
the mediawiki codesniffer config ist stored in the .phpcs.xml. it uses the rule-set from:
./vendor/mediawiki/mediawiki-codesniffer/MediaWiki

however the mediawiki rules are very strict and curretnly there are exceptions from this rules in the .phpcs.xml as well:

    <exclude name="Generic.Files.LineLength.TooLong" />
    <exclude name="Squiz.WhiteSpace.SuperfluousWhitespace.EndLine" />

    <exclude name="MediaWiki.Commenting" />

    <exclude name="MediaWiki.WhiteSpace.SpaceBeforeSingleLineComment.SingleSpaceBeforeSingleLineComment" />
    <exclude name="MediaWiki.WhiteSpace.SpaceBeforeSingleLineComment.NewLineComment" />
    <exclude name="MediaWiki.WhiteSpace.SpaceAfterClosure.NoWhitespaceAfterClosure" />

    <exclude name="MediaWiki.Classes.UnusedUseStatement.UnusedUse" />
    <exclude name="MediaWiki.Classes.UnsortedUseStatements.UnsortedUse" />

    <exclude name="MediaWiki.ControlStructures.IfElseStructure.SpaceBeforeElse" />
    <exclude name="MediaWiki.Usage.StaticClosure.StaticClosure" />
    <exclude name="MediaWiki.Usage.ExtendClassUsage.FunctionConfigUsage" />
    <exclude name="MediaWiki.Commenting.PropertyDocumentation.MissingDocumentationPrivate" />
    <exclude name="MediaWiki.NamingConventions.PrefixedGlobalFunctions.allowedPrefix" />

    <!-- This is to avoid false positives when using PHP 8.0 named arguments -->
    <exclude name="Generic.PHP.DiscourageGoto.Found" />

    <exclude name="PSR2.Classes.PropertyDeclaration.Multiple" />

These two exceptions curretnly make the tests not fail, when doc blocks are missing:
MediaWiki.Commenting
MediaWiki.Commenting.PropertyDocumentation.MissingDocumentationPrivate

Reply:
I think we should follow @JeroenDeDauw's recommendation, because it seems to me that the MW codebase does a mixture of legacy code-block and PHP type annotation. And while doing the transitioning, it needs to enforce `MediaWiki.Commenting`. We can exclusively use type annotation because DataAccounting's codebase is small.

Reply:
ok, i removed the superfluous doc blocks